test/functoria/errors/run.t

```diff
@@ -39,37 +39,32 @@ Help does not fail
          Use test help <command> for more information on a specific command.
   
   COMMANDS
-         build
+         build [OPTION]… 
```

Comment:
@dbuenzli here is an example of extra whitespace at the end of the line

Reply:
The _upstream_ version of `cmdliner` does not add extra whitespace at the end of the line.